Rubberduck.VBEditor.VBA/SafeComWrappers/Application/AccessApp.cs

```diff
@@ -14,9 +15,11 @@ public class AccessApp : HostApplicationBase<Microsoft.Office.Interop.Access.App
         private const string ReportClassName = "Access.Report";
 
         private readonly Lazy<IVBProject> _dbcProject;
+        private readonly Logger _logger;
```

Comment:
This can be static, and it can be initialized inline instead of in the ctor, _and_ I'd put it in the base class instead while you're at it - `AccessApp` isn't the only host application that should have access to a logger.

Reply:
Yeah you're absolutely right. I'll take it out and put it in base where it belongs.